lib/ansible/modules/system/service_facts.py

```diff
@@ -32,7 +32,6 @@
 
 
 author:
-  - Matthew Jones
```

Comment:
Here we are removing authors, maybe this was deliberate (in consent) but otherwise I would object to it.

Reply:
This is deliberate as the module was based on Matt's, though he doesn't maintain (not should be pinged) on this